### ISSUE TITLE
Imgix aspect ratio

### ISSUE THREAD

**Issue by robbinworks** [datocms/gatsby-source-datocms#87]
👋 

Trying to use [Aspect ratio | `ar`](https://docs.imgix.com/apis/url/size/ar) and I'm getting the following error:

`error  Expected type Float, found "16:9"`

This is what my code looks like:

```js
image {
  fluid(imgixParams: { auto: "compress", fit: "crop", w: "400", ar: "16:9" }) {
    ...GatsbyDatoCmsFluid
  }
}
```

If I change the `ar` parameter to float, e.g `ar: 16.9` the error disappear but the parameter invalid, should be `W:H` as according to [the docs](https://docs.imgix.com/apis/url/size/ar).

I've tried using both fluid and fixed images.

Is this a bug in this package or am I missing something?

Thanks,

**Comment by matjack1** [comment]
I agree with you @robbinworks :)

I think we should change this!

**Comment by stefanoverna** [comment]
Fixed in 685ba3c!
